dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -3,19 +3,35 @@
 mkdir -p /etc/frr
 mkdir -p /etc/supervisor/conf.d
 
-CFGGEN_PARAMS=" \
+CONFIG_TYPE=`sonic-cfggen -d -v 'DEVICE_METADATA["localhost"]["docker_routing_config_mode"]'`
+
+CFGGEN_PARAMS_SEPARATED=" \
     -d \
     -y /etc/sonic/constants.yml \
-    -t /usr/share/sonic/templates/supervisord/frr_vars.j2 \
     -t /usr/share/sonic/templates/supervisord/supervisord.conf.j2,/etc/supervisor/conf.d/supervisord.conf \
     -t /usr/share/sonic/templates/bgpd/bgpd.conf.j2,/etc/frr/bgpd.conf \
     -t /usr/share/sonic/templates/zebra/zebra.conf.j2,/etc/frr/zebra.conf \
     -t /usr/share/sonic/templates/staticd/staticd.conf.j2,/etc/frr/staticd.conf \
+    -t /usr/share/sonic/templates/isolate.j2,/usr/sbin/bgp-isolate \
+    -t /usr/share/sonic/templates/unisolate.j2,/usr/sbin/bgp-unisolate \
+"
+
+CFGGEN_PARAMS_SPLIT=" \
+    -d \
+    -y /etc/sonic/constants.yml \
+    -t /usr/share/sonic/templates/supervisord/supervisord.conf.j2,/etc/supervisor/conf.d/supervisord.conf \
+    -t /usr/share/sonic/templates/isolate.j2,/usr/sbin/bgp-isolate \
+    -t /usr/share/sonic/templates/unisolate.j2,/usr/sbin/bgp-unisolate \
+"
+
+CFGGEN_PARAMS_UNIFIED=" \
```

Comment:
this is unrelated change, should be move to a different pr.

Reply:
OK, I will move it to another PR. This change was done to support split mode along with separate mode. The default is still separate mode.